### PR TITLE
Updated README.MD

### PR DESCRIPTION
**Pull request by hosy**
## Description
This PR is updating the README.MD file for a better marketing on Github, because the readme file is shown the user as entry page.

## Motivation and Context
- Better Marketing for our iOS app
- Show contributors relevant informations
- Show features of our app

This PR will also add the `fastlane/screenshots/` folder to git, because with this we have the latest AppStore screenshots as backup. Furthermore the latest screenshots are used in the readme file.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by hosy** [comment]
@michaelstingl please have a look, if you want to use this new readme file for our repository:
https://github.com/owncloud/ios-app/blob/github-readme-update/README.md

File: README.md [hunk 1]
```diff
@@ -1,33 +1,68 @@
-# [ownCloud](https://owncloud.org) iOS app
+# [ownCloud](https://owncloud.org) iOS App
 
-Available in the Apple App Store:
+## 📱 The new iOS app for your ownCloud!
```
Comment (jesmrec) [inline README.md]: As the change above: app -> App

File: README.md [hunk 2]
```diff
@@ -1,33 +1,68 @@
...
 
+## 🛠 Build our App
 
-## Join development!
+To build our app, please read our [SETUP.md](https://github.com/owncloud/ios-app/blob/master/SETUP.md)
 
-**Build status:**
+```
+$ read [SETUP.md]()
+```
```
Comment (jesmrec) [inline README.md]: That script to read he SETUP.md is not correctly interpreted

Screenshot 2020-01-21 at 12 35 33

File: README.md [hunk 3]
```diff
@@ -1,33 +1,68 @@
...
+## 📲 Download
+
+Our iOS app is free for downloading available on the AppStore
 
 [![](https://owncloud.org/wp-content/themes/owncloudorgnew/assets/img/clients/buttons/appstore.png)](https://apps.apple.com/app/id1359583808)
 
+## 🛠 Build our App
 
-## Join development!
+To build our app, please read our [SETUP.md](https://github.com/owncloud/ios-app/blob/master/SETUP.md)
```
Comment (jesmrec) [inline README.md]: app -> App

File: README.md (hunk 2 above)
Comment (hosy) [reply README.md]: I removed the brackets, but it should look like a command line shell.

File: README.md (hunk 3 above)
Comment (hosy) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (jesmrec) [reply README.md]: Then it is OK

**Pull request merged**